tests/test_routes/conftest.py

```diff
@@ -0,0 +1,95 @@
+import json
+import pytest
+
+from lib.models.rocket import Rocket
+from lib.models.motor import Motor, MotorTank, TankFluids, TankKinds
+from lib.models.environment import Env
+
+
+@pytest.fixture
+def stub_env():
+    env = Env(latitude=0, longitude=0)
+    env_json = env.model_dump_json()
+    return json.loads(env_json)
```

Comment:
I might not have all the context here, but why json dump the object an then load it? I suppose this is Pydantic way of (de)serializing, but I am just confused on why perform this action at fixture level and not in a test.

Reply:
It is indeed, the reason is that we don't want to test `pydantic` implementation, so ideally we want to make it as transparent as possible for the test consuming the fixture.

My tests are expecting serialized data to work with, but if I try to simply `model_dump()` I keep getting ``` Object of type datetime is not JSON serializable```, so one way to overcome that was to dump to json and then load a dict without pydantic toolkit.